tests/shared/core/test_trackers.py

```diff
@@ -36,6 +36,8 @@
     LegacyForm,
     LegacyFormValidation,
     LoopInterrupted,
+    DefinePrevUserUtteredFeaturization,
```

Comment:
@Ghostvv @tmbo Brought up if we can come up with better names / names which generalize better:

How about 
* `UserUtteredFeaturizated`
* `EntitiesAdded`

Reply:
In the redesigned system, when/if we have them as separate events, I think it is good names. In the current system, I'd keep `Define` ones. But if you think it is better to rename now, please go for it

Reply:
@joejuzl What do you think?

Reply:
If these are the only two which behave in this way I can see an argument for having them have a common prefix.

Reply:
Tom also meant the point that all events are using the past tense to stress the fact that they mirror something in the past. So I think it makes sense to at least take that into account.

Reply:
And events are so essential that they'll basically stay around forever one they were added.  Hence, more general names could help us to adapt in the future.

Reply:
taking into account new development around e2e entities, we can safely rename them

Reply:
Do you mean we should rename them because a lot of things around entities are gonna change or did you mean there is no need to rename as things are gonna stay the same? 🙈

Reply:
I meant we can rename them because for entities we're not "defining" previous useruttered entities anymore, so the name `DefinePrevUserUtteredFeaturization` is not accurate. And since you pointed out that deprecating event names is a pain, we might go with the names we like from the start

Reply:
Ok, so how about `EntitiesAdded`?

Reply:
let's add `e2e` in there? for explicitness

Reply:
In theory it doesn't really matter where entities are coming from, no?

Reply:
Eg. if we split up events in something like 

* `UserInputEvent` -> pure unchanged user input
* `NLUIntentEvent`
* `NLUEntityEvent`
* `PolicyPrediction`

then we could actually use `EntityAdded` both for NLU and Policy prediction

Reply:
sounds good

Reply:
done